### ISSUE TITLE
Fatal throwable error on generate

### ISSUE THREAD

**Issue by rafiktoumi** [kitloong/laravel-migrations-generator#67]
Hi,
I'm on laravel 6 and after installing the package with composer and running `php artisan migrate:generate` i get this error 
`Symfony\Component\Debug\Exception\FatalThrowableError  : Argument 1 passed to MigrationsGenerator\MigrationsGeneratorSetting::setTableFilename() must be of the type string, null given, called in C:\lar
agon\www\erp\vendor\kitloong\laravel-migrations-generator\src\MigrationsGenerator\MigrateGenerateCommand.php on line 116`

**Comment by kitloong** [comment]
Hi @rafiktoumi 

Thank you for reporting. 

`setTableFilename` will either read `table-filename` from argument or default filename pattern from the built-in config file.

To help investigate, could you please:
1. Use the latest version of the generator.
2. If issue still persists, try install in a fresh new Laravel and run the generator.
3. If above not working for you, please send detail information of your `composer.json` and version number to reproduce the error.

**Comment by EXEIdeas** [comment]
Hi @kitloong,

Same here. I'm on Laravel 5.8 and after installing the package with composer and running `php artisan migrate:generate` I get the below error too...

`Symfony\Component\Debug\Exception\FatalThrowableError : Argument 1 passed to MigrationsGenerator\MigrationsGeneratorSetting::setTableFilename() must be of the type string, null given, called in C:\lar agon\www\XXXXXXXX\vendor\kitloong\laravel-migrations-generator\src\MigrationsGenerator\MigrateGenerateCommand.php on line 116`

I also cleared `$ php artisan config:clear`  but no success. I am also using the Latest Release. Here is my [composer.json](https://textdoc.co/2aCumTMU8YDsGPdf) file

**Comment by kitloong** [comment]
Hi @EXEIdeas 

Thank you for reporting.

Could you please `composer remove`
1. `oscarafdev/migrations-generator`
2. `xethron/migrations-generator` 

Then retry with the generator?

Also, I encourage you to test with a fresh new Laravel project.

**Comment by kitloong** [comment]
Hi @rafiktoumi @EXEIdeas 

It seems the error is caused by multiple generator installation conflicts.

I will close this issue for now.
However, if you have more information to share, feel free to reopen this issue.

Thank you!

**Comment by EXEIdeas** [comment]
Sorry for Late Reply. I actually tried multiple after this was not working. Anyway, I remove all other packages but still not working. Now it is giving me the below error.

First I run this command line.
`composer require --dev "kitloong/laravel-migrations-generator"`

Then edit app.php file and added the below code to register this.
```
'providers' => [
    ..................
    \MigrationsGenerator\MigrationsGeneratorServiceProvider::class,
  ],
```

Then run the below code as I have many connections to different database in my database.php. MY localhost connection is labeled as localhost also as I want to generate migrations for my localhost database tables only.
`php artisan migrate:generate --connection="localhost"`


Now after doing all above, I got the below error...
```
$ php artisan migrate:generate --connection="localhost"
In MigrationsGeneratorSetting.php line 225:
Return value of MigrationsGenerator\MigrationsGeneratorSetting::getTableFilename() must be of the type string, null returned
```
